### PR TITLE
Add missing preload_link_tag method in ActionView helper guide [ci skip]

### PR DESCRIPTION
**Pull request by aashishpsaini**
### Detail

This Pull Request adds preload_link_tag in ActionView helper method guide.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: guides/source/action_view_helpers.md [hunk 1]
```diff
@@ -99,6 +99,15 @@ javascript_url "common"
 # => http://www.example.com/assets/common.js
 ```
 
+#### preload_link_tag
+
+Returns a link tag that browsers can use to preload the source. The source can be the path of a resource managed by asset pipeline, a full path, or an URI.
+
+```ruby
+preload_link_tag("application.css")
```
Comment (vipulnsward) [inline guides/source/action_view_helpers.md:107]:
```suggestion
preload_link_tag "application.css"
```

Comment (aashishpsaini) [reply guides/source/action_view_helpers.md]: Done

**Comment by vipulnsward** [comment]
Can you squash your commits?

**Comment by aashishpsaini** [comment]
> Can you squash your commits?

Done. Thank you.

**Pull request merged**